src/appleseed/foundation/meta/tests/test_imagefilewriter.cpp

```diff
@@ -0,0 +1,217 @@
+// appleseed.foundation headers.
```

Comment:
Can you add a copyright notice on the top of the file with your name and `2020`, like we do in other files.

Reply:
copyright notice added. Thank you!